store/job_storer_test.go

```diff
@@ -0,0 +1,58 @@
+package store
+
+import (
+	"context"
+	"github.com/ONSdigital/dp-search-reindex-api/models"
+	"testing"
+	"time"
+	. "github.com/smartystreets/goconvey/convey"
+)
+
+// Constants for testing
+const (
+	testJobID1 = "UUID1"
+)
+
+var ctx = context.Background()
+
```

Comment:
As already commented are you able to confirm what the expected/required behaviour is around attempting to create a new job with an existing ID?
Once that is clarified can you add some test cases to cover those requirements?

Reply:
please refer to earlier reply - I don't think these scenarios are likely to occur since the id gets generated by the service